.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -9,10 +9,9 @@ body:
       value: |
         **Thank you for reporting a bug in Ansible Builder.**
 
-        If you are looking for community support, please reach out on IRC:
-
-          [#ansible](https://web.libera.chat/#ansible)
-          [#ansible-builder](https://web.libera.chat/#ansible-builder)
+        If you are looking for community support, please visit
+        the [Community guide](https://ansible.readthedocs.io/projects/builder/en/latest/community/)
+        for information on how to get in touch.
```

Comment:
The line beginning with "for" seems to be under indented when I download the change locally and inspect it (causing the YAML linting error you see in CI).

Reply:
@Shrews Fixed, thanks! PTAL